web_src/js/markup/anchors.js

```diff
@@ -16,7 +16,7 @@ function scrollToAnchor() {
 }
 
 export default function initMarkdownAnchors() {
```

Comment:
`initMarkupAnchors`?

Reply:
Fixed, renamed and changed these to named exports.